File: backend/src/main/java/hu/ibc/ibcpartners/partner/dto/PartnerDto.java
package hu.ibc.ibcpartners.partner.dto;

import jakarta.validation.constraints.NotBlank;

import java.util.List;
import java.util.UUID;

public record PartnerDto(
        Long id,
        @NotBlank String taxNumber,
        @NotBlank String name,
        @NotBlank String headquarters,
        String site,
        String location,
        String contact,
        String phone,
        String email,
        String website,
        List<Long> activities,
        Long referralId,
        String keyWords,
        String introduction,
        String photo,
        String logo) {
}

File: frontend/src/app/features/dashboard/partner/partner-dto.ts
export interface PartnerDto {
  id: number;
  taxNumber: string;
  name: string;
  headquarters: string;
  site: string | null;
  location: string | null;
  contact: string | null;
  phone: string | null;
  email: string | null;
  website: string | null;
  activities: number[];
  keyWords: string | null;
  introduction: string | null;
  photo: string | null;
  logo: string | null;
}
